content/patterns/checkbox/checkbox-pattern.html

```diff
@@ -14,61 +14,62 @@
         </ul>
       </section>
 
       <section id="keyboard_interaction">
-        <h2>Keyboard Interaction</h2>
-        <p>When the checkbox has focus, pressing the <kbd>Space</kbd> key changes the state of the checkbox.</p>
+        <h2>キーボードの操作</h2>
+        <p>チェックボックスにフォーカスがあるとき、<kbd>Space</kbd> キーを押すとチェックボックスの状態が変わります。</p>
       </section>
 
       <section id="roles_states_properties">
-        <h2>WAI-ARIA Roles, States, and Properties</h2>
+        <h2>WAI-ARIAのロール、ステート、及びプロパティ</h2>
         <ul>
-          <li>The checkbox has role <a href="#checkbox" class="role-reference">checkbox</a>.</li>
+          <li>チェックボックスには <a href="#checkbox" class="role-reference">checkbox</a> ロールがあります。</li>
           <li>
-            The checkbox has an accessible label provided by one of the following:
+            チェックボックスには、以下のいずれかによってアクセシブルなラベルが提供されています：
             <ul>
-              <li>Visible text content contained within the element with role <code>checkbox</code>.</li>
-              <li>A visible label referenced by the value of <a href="#aria-labelledby" class="property-reference">aria-labelledby</a> set on the element with role <code>checkbox</code>.</li>
-              <li><a href="#aria-label" class="property-reference">aria-label</a> set on the element with role <code>checkbox</code>.</li>
+              <li><code>checkbox</code> ロールを持つ要素内に含まれる表示可能なテキストコンテンツ。</li>
+              <li><code>checkbox</code> ロールを持つ要素に設定された <a href="#aria-labelledby" class="property-reference">aria-labelledby</a> の値によって参照される表示可能なラベル。</li>
+              <li><code>checkbox</code> ロールを持つ要素に設定された <a href="#aria-label" class="property-reference">aria-label</a> 。</li>
             </ul>
           </li>
-          <li>When checked, the checkbox element has state <a href="#aria-checked" class="state-reference">aria-checked</a> set to <code>true</code>.</li>
-          <li>When not checked, it has state <a href="#aria-checked" class="state-reference">aria-checked</a> set to <code>false</code>.</li>
-          <li>When partially checked, it has state <a href="#aria-checked" class="state-reference">aria-checked</a> set to <code>mixed</code>.</li>
-          <li>If a set of checkboxes is presented as a logical group with a visible label, the checkboxes are included in an element with role <a href="#group" class="role-reference">group</a> that has the property <a href="#aria-labelledby" class="property-reference">aria-labelledby</a> set to the ID of the element containing the label.</li>
-          <li>If the presentation includes additional descriptive static text relevant to a checkbox or checkbox group, the checkbox or checkbox group has the property <a href="#aria-describedby" class="property-reference">aria-describedby</a> set to the ID of the element containing the description.</li>
+          <li>チェックされている場合、チェックボックス要素の状態は <a href="#aria-checked" class="state-reference">aria-checked</a> が <code>true</code> に設定されています。</li>
```

Comment:
この state は ARIA のロール、ステート、プロパティのうちのステートですね。WCAGのほうでは「状態 (state)」としているやつです。ARIA側ではどうしてましたっけ。カタカナ「ステート」?

```suggestion
          <li>チェックされている場合、チェックボックス要素の <a href="#aria-checked" class="state-reference">aria-checked</a> ステートが <code>true</code> に設定されています。</li>
```